### PR TITLE
(Fix) Status info consistency issues

### LINKED ISSUE #342: [FE] Status Info consistency issues
While we are correctly informing _most_ of the transactions' statuses, we should make them more consistent.

1. For instance, on "Prepare Condition", complete the form, click on "Prepare", and reject the transaction in Metamask. This is the result:

Screen Shot 2020-09-24 at 15 27 55

That shouldn't be the case. Show a proper "Error" `FullLoading` component (see screenshot in next item).

**Title:** Error.
**Icon:** Error icon.
**Text:** Same text as in the screenshot.
**Button:** Red "Close" button.

2. For "Split Position", complete the form, click on "Split", and reject the transaction in Metamask. This is the result:

Screen Shot 2020-09-24 at 15 28 30

This is mostly OK, but anyways:

**Title:** Error.
**Icon:** Error icon.
**Text:** Same text as in the screenshot.
**Button:** Red "Close" button.

Also modal's width should be narrower (see screenshot in item `#3`, it should look like it).

3. For "Merge Positions", complete the form, click on "Merge", and reject the transaction in Metamask. This is the result:

Screen Shot 2020-09-24 at 15 30 34

It's OK, but the button should be red.

**Title:** Error.
**Icon:** Error icon.
**Text:** Same text as in the screenshot.
**Button:** Red "Close" button.

4. For "Redeem Positions", complete the form, click on "Redeem", and reject the transaction in Metamask. This is the result:

Screen Shot 2020-09-24 at 15 33 30

It's OK, but the button should be red.

**Title:** Error.
**Icon:** Error icon.
**Text:** Same text as in the screenshot.
**Button:** Red "Close" button.

5. For "Report Payouts", complete the form, click on "Report", and reject the transaction in Metamask. This is the result:

Nothing. There's no error.

So, implement this:

**Title:** Error.
**Icon:** Error icon.
**Text:** Meaningful text.
**Button:** Red "Close" button.

### PR DESCRIPTION
**Pull request by gabitoesmiapodo**
Closes #342

Not adding screenshots because it's a lot of extra work, but the status info modal windows all should look as requested now.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr442--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr442--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr442--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr442--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr442--conditionaltokensexplorer.review.gnosisdev.com

**Review by hpmaxi — CHANGES_REQUESTED** [review]

File: src/components/text/FormatHash/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+interface Props extends HTMLAttributes<HTMLSpanElement> {
+  hash: string
+}
+
+export const FormatHash: React.FC<Props> = (props) => {
+  const { hash, ...restProps } = props
+
+  const cleanHash = hash?.toString().toLowerCase().replace('0x', '')
+
```
[the remark this answers is not in the document]
Comment (gabitoesmiapodo) [reply src/components/text/FormatHash/index.tsx]: I don't think it should affect anything, it's only used to display addresses.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr442--conditionaltokensexplorer.review.gnosisdev.com

**Review by hpmaxi — APPROVED** [review]

**Review by mariano-aguero — APPROVED** [review]

**Pull request merged**